atomistics/calculators/sphinxdft.py

```
@@ -0,0 +1,136 @@
+        "scf_energy_int"
+    ][-1][-1]
+    return energy_scf_int, forces
+
+
+def calc_static_with_sphinxdft(
+    structure: Atoms,
+    working_directory: str,
+    executable_function: callable,
+    maxSteps: int = 100,
+    eCut: float = 25,
+    kpoint_coords: list[float, float, float] = [0.5, 0.5, 0.5],
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)